src/tools/web/client/README.md

```diff
@@ -0,0 +1,17 @@
+# elektra-web client
+
+
+## Installation
+
+```
+npm install
```

Comment:
> In which directory?

all commands should be run in the same directory as the README

> Tell where the dependencies are listed.

In `package.json` - this is a nodejs standard

> Installation fails with:

I just tried reinstalling and it worked fine for me. It might be your node version, which is very old (current release is 6.x)

> cmake integration?

should be possible, it just needs to execute some bash commands